src/code/code_80097A00.c

```diff
@@ -78,136 +78,141 @@ u32 gGsFlagsMasks[] = { 0x000000FF, 0x0000FF00, 0x00FF0000, 0xFF000000 };
+    gItemIconTunicZoraTex,           // ITEM_TUNIC_ZORA
+    gItemIconBootsKokiriTex,         // ITEM_BOOTS_KOKIRI
+    gItemIconBootsIronTex,           // ITEM_BOOTS_IRON
+    gItemIconBootsHoverTex,          // ITEM_BOOTS_HOVER
+    gItemIconBulletBag30Tex,         // ITEM_BULLET_BAG_30
+    gItemIconBulletBag40Tex,         // ITEM_BULLET_BAG_40
+    gItemIconBulletBag50Tex,         // ITEM_BULLET_BAG_50
+    gItemIconQuiver30Tex,            // ITEM_QUIVER_30
+    gItemIconQuiver40Tex,            // ITEM_QUIVER_40
+    gItemIconQuiver50Tex,            // ITEM_QUIVER_50
+    gItemIconBombBag20Tex,           // ITEM_BOMB_BAG_20
+    gItemIconBombBag30Tex,           // ITEM_BOMB_BAG_30
+    gItemIconBombBag40Tex,           // ITEM_BOMB_BAG_40
+    gItemIconGoronsBraceletTex,      // ITEM_STRENGTH_GORONS_BRACELET
+    gItemIconSilverGauntletsTex,     // ITEM_STRENGTH_SILVER_GAUNTLETS
+    gItemIconGoldenGauntletsTex,     // ITEM_STRENGTH_GOLD_GAUNTLETS
+    gItemIconScaleSilverTex,         // ITEM_SCALE_SILVER
+    gItemIconScaleGoldenTex,         // ITEM_SCALE_GOLDEN
+    gItemIconBrokenGiantsKnifeTex,   // ITEM_SWORD_KNIFE
+    gItemIconAdultsWalletTex,        // ITEM_ADULTS_WALLET
+    gItemIconGiantsWalletTex,        // ITEM_GIANTS_WALLET
+    gItemIconDekuSeedsTex,           // ITEM_DEKU_SEEDS
+    gItemIconFishingPoleTex,         // ITEM_FISHING_POLE
+    // icon_item_static 16x24 ia8
+    gSongNoteTex, // ITEM_SONG_MINUET
+    gSongNoteTex, // ITEM_SONG_BOLERO
+    gSongNoteTex, // ITEM_SONG_SERENADE
+    gSongNoteTex, // ITEM_SONG_REQUIEM
+    gSongNoteTex, // ITEM_SONG_NOCTURNE
+    gSongNoteTex, // ITEM_SONG_PRELUDE
+    gSongNoteTex, // ITEM_SONG_LULLABY
+    gSongNoteTex, // ITEM_SONG_EPONA
+    gSongNoteTex, // ITEM_SONG_SARIA
+    gSongNoteTex, // ITEM_SONG_SUN
+    gSongNoteTex, // ITEM_SONG_TIME
+    gSongNoteTex, // ITEM_SONG_STORMS
+    // icon_item_24static 24x24 rgba32
+    gQuestIconForestMedallionTex, // ITEM_MEDALLION_FOREST
+    gQuestIconFireMedallionTex,   // ITEM_MEDALLION_FIRE
+    gQuestIconWaterMedallionTex,  // ITEM_MEDALLION_WATER
+    gQuestIconSpiritMedallionTex, // ITEM_MEDALLION_SPIRIT
+    gQuestIconShadowMedallionTex, // ITEM_MEDALLION_SHADOW
+    gQuestIconLightMedallionTex,  // ITEM_MEDALLION_LIGHT
+    gQuestIconKokiriEmeraldTex,   // ITEM_SPSTONE_KOKIRI_EMERALD
+    gQuestIconGoronRubyTex,       // ITEM_SPSTONE_GORON_RUBY
+    gQuestIconZoraSapphireTex,    // ITEM_SPSTONE_ZORA_SAPPHIRE
+    gQuestIconStoneOfAgonyTex,    // ITEM_STONE_OF_AGONY
+    gQuestIconGerudosCardTex,     // ITEM_GERUDOS_CARD
+    gQuestIconGoldSkulltulaTex,   // ITEM_SKULL_TOKEN
+    gQuestIconHeartContainerTex,  // ITEM_HEART_CONTAINER
+    gQuestIconHeartPieceTex,      // ITEM_HEART_PIECE
+    gQuestIconDungeonBossKeyTex,  // ITEM_DUNGEON_BOSS_KEY
+    gQuestIconDungeonMapTex,      // ITEM_DUNGEON_COMPASS (swapped with map?)
+    gQuestIconDungeonCompassTex,  // ITEM_DUNGEON_MAP
+    gQuestIconDungeonBossKeyTex,  // ITEM_SMALL_KEY
```

Comment:
Huh, it uses boss key for both, or...?

Reply:
I can't tell if any of these 4 actually uses this array to get the icon, it doesn't seem like it but idk. At least `ITEM_SMALL_KEY` I think is never used in pause?

Reply:
Added comments on the three odd item->texture mappings (map, compass, small key)
It's inconsequential for all three (as `gItemIcons` isn't accessed for those items)